### ISSUE TITLE
bug: emojis break alignments

### ISSUE THREAD

**Issue by mashehu** [ewels/rich-codex#34]
My guess is that there are two different widths of emojis used during placement and then rendering:
<img width="767" alt="Screenshot 2022-08-18 at 11 11 55" src="https://user-images.githubusercontent.com/6169021/185358478-b856477a-ad83-4bf3-b50f-84e43f51fc90.png">

**Comment by ewels** [comment]
oof, that's going to be difficult / impossible to fix I think.. Suspect that it's an upstream issue.

**Comment by mashehu** [comment]
Yeah, not expecting a fix soon. Just wanted to report it. I know that variable emoji-widths are one of their big drawbacks.

**Comment by ewels** [comment]
Should now, finally, be fixed in https://github.com/ewels/rich-codex/pull/66
